src/ir/context.rs

```diff
@@ -695,6 +695,18 @@ impl<'ctx> BindgenContext<'ctx> {
             CXType_Float => TypeKind::Float(FloatKind::Float),
             CXType_Double => TypeKind::Float(FloatKind::Double),
             CXType_LongDouble => TypeKind::Float(FloatKind::LongDouble),
+            CXType_Complex => {
+                let float_type = ty.elem_type()
+                    .expect("Not able to resolve complex type?");
+                let float_kind = match float_type.kind() {
```

Comment:
probably: 

```rust
let float_kind = match float_type.kind() {
    CXType_Float => FloatKind::Float,
    CXType_Double => FloatKind::Double,
    CXType_LongDouble => FloatKind::LongDouble,
    _ => panic!("Non floating-type complex?"),
};
```

Or, you could also add a `FloatKind::from_ty(Type) -> Option<FloatKind>` function, though that might be overkill for this, so leaving a `match` is totally fine :)